packages/cms-data-sync/test/fixtures/contentful-environment.fixtures.ts

```diff
@@ -0,0 +1,101 @@
+import { Environment, Link, SysLink } from 'contentful-management';
+
+export const contenfulSpaceLink: SysLink = {
+  sys: { type: 'Link', linkType: 'Space', id: 'space-id' },
+};
+
+export const contentfulEnvironmentLink: SysLink = {
+  sys: { id: 'env-id', type: 'Link', linkType: 'Environment' },
+};
+
+export const contenfulUserLink: Link<'User'> = {
+  sys: { type: 'Link', linkType: 'User', id: 'user-id' },
+};
+
+const baseEnvironment: Environment = {
```

Comment:
this is not really a fixture, more like a mock no? maybe move to `mocks/`?

Reply:
committed here https://github.com/yldio/asap-hub/pull/2290/commits/ea48e78be5f28babe4bad30b406bb7466506473e